packages/client/hmi-client/src/components/workflow/ops/interventions/tera-intervention-card.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+	<div class="intervention-card">
+		<header class="flex align-items-center">
+			<tera-edit-value
+				class="mr-auto"
+				:model-value="intervention.name"
+				@update:model-value="onUpdateName($event)"
+				tag="h5"
+			/>
+			<RadioButton
+				:model-value="interventionType"
+				:input-id="uniqueId()"
+				name="interventionType"
+				value="static"
+				@click="onInterventionTypeChange('static')"
+			/>
+			<label for="static" class="ml-2">Static</label>
+			<RadioButton
```

Comment:
Is this dynamic intervention type able to be used by anything?
Should we disable this button for now?

Reply:
I agree

Reply:
As discussed earlier, I think we still want the ability to make dynamic interventions, its a question of whether or not we want to use them in other node where we can just filter them out